servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/GraphQLRoutesConfiguration.kt

```diff
@@ -45,8 +45,9 @@ class GraphQLRoutesConfiguration(
 
         (isEndpointRequest and isNotWebSocketRequest).invoke { serverRequest ->
             val graphQLResponse = graphQLServer.execute(serverRequest)
+            val accept = serverRequest.headers().accept().find { it == MediaType.APPLICATION_JSON } ?: MediaType.APPLICATION_GRAPHQL_RESPONSE
```

Comment:
since `application/json` is current behavior I'd keep it as default for the next release as well i.e. 
```suggestion
            val acceptMediaType = serverRequest.headers().accept().find { it.includes(MediaType.APPLICATION_GRAPHQL_RESPONSE) }?.let { MediaType.APPLICATION_GRAPHQL_RESPONSE } ?: MediaType.APPLICATION_JSON
```

Reply:
Thank you your review!
I've changed as the above.